### DISCUSSION TITLE
Do not support  ILI9948 touch driver is ft6236 on esp32 ? would you have a plan to add it

### DISCUSSION THREAD

**Discussion by sushipan** [Bodmer/TFT_eSPI#2693] [New processor/display support or capability request]
1.Do not support  ILI9948 touch driver is ft6236 on esp32 ? would you have a plan to add it?
2. IDE: PlatformIO
3. support on this repo https://github.com/DustinWatts/FT6236
4. TFT_eSPI library version "2.5.30"
5. Procesor,  ESP32
6. TFT driver (e.g. ILI9948)
7. Interface type (SPI or parallel)
8. Touch Interface is i2c
9. Screen is 3.5inch_SPI_Module_MSP3520
lcdwiki : http://www.lcdwiki.com/3.5inch_SPI_Module_ILI9488_SKU:MSP3520

**Comment by Bodmer** [comment]
Capacitance and I2C bus touch controllers are not supported. You should be able to use Dustin Watts library or another one. For example there is [one here.](https://github.com/lewisxhe/FocalTech_Library)

I have created a [library here for the FT5436 controller](https://github.com/Bodmer/Touch_FT5436) but I suspect it will not work with the FT6236

The ILI9488 is already supported.

**Comment by yel-best** [comment]
> 1.Do not support ILI9948 touch driver is ft6236 on esp32 ? would you have a plan to add it? 2. IDE: PlatformIO 3. support on this repo https://github.com/DustinWatts/FT6236 4. TFT_eSPI library version "2.5.30" 5. Procesor, ESP32 6. TFT driver (e.g. ILI9948) 7. Interface type (SPI or parallel) 8. Touch Interface is i2c 9. Screen is 3.5inch_SPI_Module_MSP3520 lcdwiki : http://www.lcdwiki.com/3.5inch_SPI_Module_ILI9488_SKU:MSP3520

I found that when using FT6236, there will be screen contact offset, and there is no way to click the button correctly.

Is there any solution? thanks

```
#include <FT6236.h>

FT6236 ts = FT6236();

static const uint16_t screenWidth = 480;
static const uint16_t screenHeight = 320;

tft.setRotation(3);

if (!ts.touched()) {
    data->state = LV_INDEV_STATE_REL;
  } else {
    data->state = LV_INDEV_STATE_PR;

    TS_Point p = ts.getPoint();
    /*Set the coordinates*/
    data->point.x = p.x;
    data->point.y = p.y;


    Serial.print("Data x ");
    Serial.println(p.x);

    Serial.print("Data y ");
    Serial.println(p.y);
  }
```
